src/styles/images/index.md.njk

```diff
@@ -59,7 +59,9 @@ Do not write alt text that:
 
 {{ example({group: "styles", item: "images", example: "alt-text", html: true, open: true, size: "l"}) }}
 
-Avoid images that contain text, especially PNGs and JPGs because the text becomes unreadable when the user zooms in. If the image does contain text, use SVG because it remains readable when the user zooms in. The alt text should be the same as the text displayed in the image.
+### Avoid images that contain text
+
+Text becomes unreadable when the user zooms into images, especially PNGs and JPGs. If the image does contain text, use SVG because it remains readable when the user zooms in. The alt text should be the same as the text displayed in the image.
```

Comment:
I'm concerned people might not understand SVG (vector) is a fundamentally different filetype to PNGs and JPGs (bitmap). So someone who doesn't really understand the underlying difference could paste their bitmap into an SVG, which doesn't achieve anything. I dont have a better suggestion right now.. just sharing thinking

Reply:
@joelanman This came up when we did a show and tell with the team, and it sounds like we'd be interested in adding a section on using good images in the future.

Reply:
```suggestion
Text becomes unreadable when the user zooms into images, especially PNGs and JPGs. If you need to use an image that does contain text, create it as a vector graphic (such as an SVG) so that it remains readable when the user zooms in. The alt text should be the same as the text displayed in the image.
```